perlbrew-install

```diff
@@ -1,20 +1,29 @@
 #!/bin/sh
 
-if [ "x${PERLBREWURL}" == "x" ]; then
+
+clean_exit () {
+    [ -f $LOCALINSTALLER ] && rm $LOCALINSTALLER
+    exit $1
+}
+
+LOCALINSTALLER=$(mktemp perlbrew.XXXXXX)
```

Comment:
This creates a file in the current directory (at least in GNU mktemp); see https://www.gnu.org/software/coreutils/manual/html_node/mktemp-invocation.html#mktemp-invocation .